tests/libnet/ipv6.go

```diff
@@ -12,42 +12,72 @@ import (
 )
 
 func configureIPv6OnVMI(vmi *v1.VirtualMachineInstance) error {
-
-	hasEth0Iface := func() bool {
-		err := console.RunCommand(vmi, "ip a | grep -q eth0", 30*time.Second)
+	alreadyConfigured := func() bool {
+		err := console.RunCommand(vmi, " ip a show lo | grep UP", 30*time.Second)
```

Comment:
No, it is `lo` intentionally. Since we may have a VM with no eth0 at all (in case `AutoattachPodInterface` is false and the spec has no interfaces).
Since the `if` checking `alreadyConfigured` also checking if `AutoattachPodInterface` is false, we may use `eth0`. But I prefer to keep `lo` to be on the safe side.